Automattic/jetpack
# Gutenpack: Add/contact form to the gutenberg editor

Fixes #9784

Simplifies the work done by @georgestephanis in https://github.com/Automattic/jetpack/pull/10256

By removing the js code since it should be automatically be included as part of the jetpack blocks. 
Now found here https://github.com/Automattic/wp-calypso/pull/27996 

#### Changes proposed in this Pull Request:


### Todo: 
[ ] Rendering on the front-end (waiting on externals) - 
https://github.com/WordPress/gutenberg/pull/11334/  

#### Testing instructions:
* Go to https://jurassic.ninja/create/?gutenberg&gutenpack&shortlived&jetpack-beta&calypsobranch=add/gutenblocks/contact-form&branch=add/contact-form-gutenblock-sdk

1. Connect Jetpack 
2. Visit the editor. Add the new contact form. 
3. Does it work as expected? 


#### Proposed changelog entry for your changes:
Add the contact form to the Gutenberg editor.
